src/java/main/org/apache/zookeeper/server/ServerStats.java

```diff
@@ -35,7 +35,7 @@
     private long totalLatency = 0;
     private long count = 0;
     private AtomicLong fsyncThresholdExceedCount = new AtomicLong(0);
-
+    
```

Comment:
nit: remove tailing white space.

Reply:
Sorry for mention this again, but I think removing the tailing white space will make the code cleaner.

Reply:
@lvfangmin seems that it doesn't have any tailing space?Am I missing something?